### PR TITLE
Revert Support for rolling upgrade of ORSet[ActorRef], #25336

### PR DESCRIPTION
**Pull request by patriknw**
This reverts commit b72312d428223156c10130e19f921e8ae5f336b6.

It has been included in several patch releases now so the migration is possible.

Refs #25336

### CODE REVIEW

**Review by chbatey — APPROVED** [review]
LGTM do we need something in the docs saying which version skip you can't do?

**Comment by patriknw** [comment]
@chbatey I have added the rolling update page here

File: akka-docs/src/main/paradox/project/rolling-update.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Rolling Updates
```
Comment (chbatey) [inline akka-docs/src/main/paradox/project/rolling-update.md]: Great

**Review by chbatey — APPROVED** [review]
Double LGTM

**Review by raboof — APPROVED** [review]
Nice cleanup, good to have a separate page for considerations for rolling upgrades, let's remember to mention this prominently in the release notes.

**Pull request merged**